modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -167,13 +163,12 @@ const editFocusedCustomObjectField = ({
 		}),
 	};
 
-	if (localizableProperty) {
-		propertyValue = localizedValue;
-	}
+	const newFocusedCustomObjectField = dataLayoutBuilder.getDataDefinitionField(
```

Comment:
@MateusCastro i noted that this ```dataLayoutBuilder.getDataDefinitionField``` is the same function used outer of this Function, to be more precise: at line 386.

Have you tried to simply return the newSettingsContext to be used in this getDataDefinitionField at line 386?

I think, maybe this Function **editFocusedCustomObjectField** could be removed and adapt or create a new one to handle this kind of change. What do you think about it, @interaminense ?

Reply:
@kevenleone, you are right. I will review the function and refactor the code.

Thanks.